content/pt-br/docs/setup/production-environment/tools/kops.md

```diff
@@ -0,0 +1,202 @@
+
+Você pode e provavelmente deve usar subdomínios para dividir seus clusters. Como nosso exemplo usaremos
+`useast1.dev.example.com`.  O endpoint do servidor de API será então `api.useast1.dev.example.com`.
+
+Uma zona hospedada do Route53 pode servir subdomínios. Sua zona hospedada pode ser `useast1.dev.example.com`,
+mas também `dev.example.com` ou até `example.com`.  kops funciona com qualquer um deles, então normalmente você escolhe por motivos de organização (por exemplo, você tem permissão para criar registros em `dev.example.com`,
+mas não em `example.com`).
+
+Vamos supor que você esteja usando `dev.example.com` como sua zona hospedada.  Você cria essa zona hospedada usando o [processo normal](https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/CreatingNewSubdomain.html), ou
+com um comando como `aws route53 create-hosted-zone --name dev.example.com --caller-reference 1`.
+
+Você deve então configurar seus registros NS no domínio principal, para que os registros no domínio sejam resolvidos. Aqui, você criaria registros NS no `example.com` para `dev`.  Se for um nome de domínio raiz, você configuraria os registros NS em seu registrador de domínio (por exemplo `example.com`,  precisaria ser configurado onde você comprou `example.com`).
+
+Verifique a configuração do seu domínio route53 (é a causa número 1 de problemas!). Você pode verificar novamente se seu cluster está configurado corretamente se tiver a ferramenta dig executando:
+
+`dig NS dev.example.com`
+
+Você deve ver os 4 registros NS que o Route53 atribuiu à sua zona hospedada.
+
+### (3/5) Crie um bucket do S3 para armazenar o estado dos clusters
+
+O kops permite que você gerencie seus clusters mesmo após a instalação. Para fazer isso, ele deve acompanhar os clusters que você criou, juntamente com suas configurações, as chaves que estão usando etc. Essas informações são armazenadas em um bucket do S3. As permissões do S3 são usadas para controlar o acesso ao bucket.
+
+Vários clusters podem usar o mesmo bucket do S3 e você pode compartilhar um bucket do S3 entre seus colegas que administram os mesmos clusters - isso é muito mais fácil do que transmitir arquivos kubecfg. Mas qualquer pessoa com acesso ao bucket do S3 terá acesso administrativo a todos os seus clusters, portanto, você não deseja compartilhá-lo além da equipe de operações.
+
+Portanto, normalmente você tem um bucket do S3 para cada equipe de operações (e geralmente o nome corresponderá ao nome da zona hospedada acima!)
+
+Em nosso exemplo, escolhemos `dev.example.com` como nossa zona hospedada, então vamos escolher `clusters.dev.example.com` como o nome do bucket do S3.
+
+* Exportar `AWS_PROFILE` (se precisar selecionar um perfil para que a AWS CLI funcione)
```

Comment:
Minor: you used the imperative tense for the other bullet points here, except this one. It would be better to use in all of them for consistency.